framework/Sources/VGSFramework/UIElements/Text Field/Validation/Date+cvv/VGSValidation.swift

```diff
@@ -9,15 +9,16 @@
 import Foundation
 
 internal class VGSValidation {
-    var pattern: String = ""
+    var regex: String = ""
+    var isLongDateFormat = false
     
     func isValid(_ txt: String, type: FieldType) -> Bool {
         if type == .none { return true }
         
-        guard txt.count != 0, pattern.count != 0 else {
+        guard txt.count != 0, regex.count != 0 else {
```

Comment:
Need to discuss this point. I mean point about default validation. 
If txt is empty and this field is not required then mb this field is valid.

Reply:
Currently we don't support any fieldType that can be empty. I mean all fildTypes has its regex that require at list few symbols.